app/src/components/Header.tsx

```
@@ -0,0 +1,202 @@
+                  <div class="p-1">
+                    <div class="text-xs font-medium text-gray-500 dark:text-gray-400 px-3 py-2 uppercase tracking-wider">
+                      Sample Contracts
+                    </div>
+                    {props.sampleContracts.map((contract) => (
+                      <button
+                        onClick={() => {
+                          props.setBytecode(contract.bytecode);
+                          props.setShowSample(false);
+                        }}
+                        class="w-full text-left px-3 py-2 text-sm text-gray-700 dark:text-gray-200 hover:bg-gray-50 dark:hover:bg-gray-800/60 rounded-md transition-colors flex flex-col"
+                      >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)